### PR TITLE
Email-3: Make gem structure, implement Google source and extracted emails

### LINKED ISSUE #3: Email: Implement Google source and extracted emails

### PR DESCRIPTION
**Pull request by dasuos**
Fixes #3

### CODE REVIEW

**Comment by klapuch** [comment]
- `test` nebo `tests`?
- lib/email/extracted_emails.rb:12 -- udelat jako lokalni promennou?
- lib/source/google_language.rb:15 -- pridat info o spatnem jazyku?
- lib/source/google_source.rb:19 -- pridat a rotovat user agenty
- rozdelit testy na unit/integration?
- Neudelat spis tridu GoogleSource, ktera by vracela jen obsah cele stranky a pak dalsi tridu, ktera by tento obsah vzala a vratila emaily podle regexu? Muze byt web, kde si muzeme chtit brat emaily podle jinych pravidel a ne podle regexu, treba podle xpath.

File: lib/email/extracted_emails.rb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    def email_domain(domain)
+      "@#{domain.sub("@", "")}"
+    end
+
+    def list_from_body(body)
+      body.scan(/\b[A-Z0-9._-]+@[A-Z0-9.-]+\.[A-Z]{2,4}\b/i)
+    end
+
+    def body(response)
+      Nokogiri::HTML(response.body).at("body").inner_html
```
Comment (dasuos) [inline lib/email/extracted_emails.rb:37]: Search for `data-header-feature` (header) and `data-content-feature` (content)  in `#search` div

File: lib/email/extracted_emails.rb [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        list_from_body(body(response))
+      end
+    end
+
+    def email_domain(domain)
+      "@#{domain.sub("@", "")}"
+    end
+
+    def list_from_body(body)
+      body.scan(/\b[A-Z0-9._-]+@[A-Z0-9.-]+\.[A-Z]{2,4}\b/i)
```
Comment (dasuos) [inline lib/email/extracted_emails.rb:33]: Restrict regular expression for the searched domain only

File: .rubocop.yml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  Enabled: false
+
+Security/Open:
+  Enabled: true
+
+Lint/PercentStringArray:
+  Enabled: false
+
+Layout/LineLength:
+  Enabled: false
```
Comment (dasuos) [inline .rubocop.yml:22]: Add newline here

**Pull request merged**